### ISSUE TITLE
PHP 8.0 deprecation - required parameters following optional

### ISSUE THREAD

**Issue by sapphirecat** [zetacomponents/Base#10]
I came across these warnings when I went digging through my `vendor` directory for any possible PHP 8 issues:

```
PHP Deprecated:  Required parameter $callback follows optional parameter $includeFilters in base/src/file.php on line 130
PHP Deprecated:  Required parameter $callbackContext follows optional parameter $includeFilters in base/src/file.php on line 130
No syntax errors detected in base/src/file.php
```

"composer show" reports that this is zetacomponents/base 1.9.1.

I'm not sure how relevant this actually is to my day-to-day life; I am using `zetacomponents/mail` to construct messages that I pass to `mailgun/mailgun-php` for actual delivery.  But the components have been absolutely wonderful to use, and I thought you'd like to know.

**Comment by derickr** [comment]
I do like to know this and we'll get it fixed!

**Comment by beberlei** [comment]
Fixed in 0a2821f0
